components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -259,15 +259,15 @@ function RepositoryNotFoundView(p: { error: StartWorkspaceError }) {
 
       if (!userScopes.includes(missingScope)) {
         setStatusMessage(<div className="mt-2 flex flex-col space-y-8">
-          <p className="text-base text-gray-400 w-96">The repository '{`${repoFullName}`}' may be private. Please authorize Gitpod to access to private repositories.</p>
+          <p className="text-base text-gray-400 w-96">The repository may be private. Please authorize Gitpod to access to private repositories.</p>
```

Comment:
```suggestion
          <p className="text-base text-gray-400 w-96">The repository <code>${repoName}</code> may be private. Please authorize Gitpod to access to private repositories.</p>
```

Reply:
Care to elaborate on why?

I reckon that keeping it inside the error message makes it pass as "unimportant" and makes also easily misread.
In the new proposed position the repo name is front and center, bigger, red: meaning that next time, the next user won't likely have the same hiccup @ghuntley lamented about in #7191 😄

Reply:
Just a more informative way to display to the users, maybe!?

Reply:
I forgot to mention that keeping the repo name also in the error message looked bad both to me and @iQQBot . See https://github.com/gitpod-io/gitpod/pull/7302#issuecomment-997430263